app/views/posts/index.html.erb

```diff
@@ -1,2 +1,24 @@
-<h1>Posts</h1>
-<p>Here is the list of Posts</p>
+<div class="main-container">
+    <%= render partial: 'partials/userinfo', locals: { user: @user} %>
+</div>
+<div class="posts-container">
+  <% @posts.each do |post| %>
+    <%= link_to user_post_path(@user, post), id: "user_#{post.id}", class: 'posts-container' do %> 
+      <%= render partial: 'partials/post', locals: { post: post } %>
+    <% end %>
+    <ul class="comments-container">
+      <% if post.comments.present? %>
+        <% recent_comments = post.comments.order(created_at: :desc).limit(2) %>
+        <% recent_comments.each do |comment| %> 
+          <li class="user-comment"><%= comment.user.name %>: <%= comment.text %></li>
+        <% end %>
+      <% else %>
+        <li class="user-comments">No comments for this post.</li>
+      <% end %>
+    </ul>
+
+  <% end %>
+</div>
+<div class="pagination">
+  <%= link_to 'Pagination', user_posts_path(@user, page: @posts.current_page + 1), class: 'pagination-btn' %>
+</div>
```

Comment:
- **[OPTIONAL]** Pagination should let us see the rest of the post. When I clicked on the button it continued showing the same quantity of posts and also they should be three as required in the model method but it shows two, the same with the comments it should show 5 comments but it shows two. Please fix this issue for a better UX 👇🏽 

![image](https://github.com/ometman/blogtalk/assets/58642949/a0498bb8-2811-4bc4-bf94-3ff52d26e7c0)

Reply:
Oh okay. Rectified. 😄